package.json

```diff
@@ -86,7 +85,8 @@
     "@types/jest": "^29.5.1",
     "@types/jest-axe": "^3.5.5",
     "@types/node": "^20.2.3",
-    "@types/nunjucks": "^3.2.2"
+    "@types/nunjucks": "^3.2.2",
+    "puppeteer": "^20.2.0"
```

Comment:
I've moved the project-level `"puppeteer"` to `optionalDependencies` as it's only used for JSDoc types